### PR TITLE
install satellizer for google auth and authentication protocol

### LINKED ISSUE #54: As a meeting initiatior, I want to be able to log in via google through the website

### PR DESCRIPTION
**Pull request by thejinxters**
closes #54

### CODE REVIEW

File: app/js/app.coffee [hunk 1]
```diff
@@ -26,11 +27,23 @@ angular.module 'tandemApp', [
...
+  $authProvider.storageType = 'localStorage'
 
-.run (AuthToken, $location) ->
+  # Auth Providers
+  $authProvider.google
+    url: CONST.API_URL + "login"
+    access_type: "offline"
+    scope: [
+      'profile'
+      'email'
```
Comment (Zandrr) [inline app/js/app.coffee]: do we need email, or just profile and cal?

File: app/js/controllers/logout_controller.coffee [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
 angular.module 'tandemApp'
 
-.controller 'LogoutController', ($location, AuthToken, Session) ->
-  Session.logout()
-  AuthToken.setToken()
-  $location.url('login')
+.controller 'LogoutController', ($location, $auth) ->
+  # Remove auth token and redirect to login
+  $auth.removeToken()
+  $location.path 'login'
```
Comment (Zandrr) [inline app/js/controllers/logout_controller.coffee]: should logout go to 'login'?

File: tests/unit/controllers/login_controller_spec.coffee [hunk 3]
```diff
@@ -20,25 +20,25 @@ describe 'Controller: LoginController', ->
...
-      # Mock call to add Mentor to server
-      SessionMock.login = (credentials) ->
-        deferred = $q.defer()
-        deferred.reject({error:"error"})
-        return deferred.promise
+  #     # Mock call to add Mentor to server
+  #     SessionMock.login = (credentials) ->
+  #       deferred = $q.defer()
+  #       deferred.reject({error:"error"})
+  #       return deferred.promise
```
Comment (Zandrr) [inline tests/unit/controllers/login_controller_spec.coffee]: Delete this unless we need it

**Comment by Zandrr** [comment]
added some nits, looks good otherwise

File: app/js/app.coffee (hunk 1 above)
Comment (thejinxters) [inline app/js/app.coffee]: update: we need the email if we want their email address

**Pull request merged**